### DISCUSSION TITLE
Does gdata-provider support tasks ordering by 'My order'?

### DISCUSSION THREAD

**Discussion by GerardEarley** [kewisch/gdata-provider#702] [Support] [answered]
I order my days tasks on Google Tasks by their importance to me and would like have the order I set appear within Mozilla Thunderbird, so I'm wondering if gdata-provider and Thunderbird support ordering with the "My order" option that appears within google tasks web version?

Many thanks

![Untitled-1](https://github.com/kewisch/gdata-provider/assets/10418107/ab95376a-b7bd-43fc-869a-596addee4489)

**Comment by kewisch** [comment, marked as answer]
Thunderbird doesn't support manual sorting, so I unfortunately can't replicate it in the add-on. I haven't checked, but I wouldn't be surprised if the API also didn't provide the order sort key.
